OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -106,6 +106,9 @@ public ConnectionFailedLogic(Widget widget, OrderManager orderManager, Action on
 			var connectionError = widget.Get<LabelWidget>("CONNECTION_ERROR");
 			connectionError.GetText = () => orderManager.ServerError;
 
+			var panelTitle = widget.Get<LabelWidget>("TITLE");
+			panelTitle.GetText = () => orderManager.AuthenticationFailed ? "Password Required" : "Connection Failed";
```

Comment:
Can you please remove the text from the yaml file, so its only defined once here?